roles/webots/behaviour.role

```diff
@@ -35,6 +35,8 @@ nuclear_role(
   skill::GetUp
   skill::Look
   skill::Dive
+  skill::Say
+  skill::GPT
```

Comment:
Why do we want this functionality in webots? Does it provide audio interfaces as well?

Reply:
Audio interface would just be device on your laptop when running in Docker, I was just using behaviour.role as a quick and easy way to test things